### ISSUE TITLE
steam.resolve()  *issue*

### ISSUE THREAD

**Issue by bg1337** [xDimGG/node-steamapi#21]
Hi,

when trying to resolve any steam profile with a custom id contain (-), the function ignores the (-)

For Example:
when resolving "https://steamcommunity.com/id/draft-"
resulted ID is : "76561198156050531" which below to a different steam profile "https://steamcommunity.com/id/draft"

**Comment by Scui1** [comment]
Thats true, There is a fix for this but I had no time to make a pull request yet
Change this line:
`const reProfileID = RegExp(String.raw`${reProfileBase}(?:id\/)?(\w{2,32})`, 'i');` [https://github.com/xDimGG/node-steamapi/blob/master/src/SteamAPI.js#L21](url)

To this line:
`const reProfileID = RegExp(String.raw`${reProfileBase}(?:id\/)?([a-zA-Z0-9_\-]{2,32})`, 'i');`

**Comment by bg1337** [comment]
didnt work

**Comment by Scui1** [comment]
Works fine for me. Just use the pull request i created, or just wait for xDimGG to deliver a fix.

**Comment by bg1337** [comment]
this what worked for me:

const reProfileID = RegExp(String.raw`${reProfileBase}(?:id\/)?([a-zA-Z0-9_-]{2,32})`, 'i');

**Comment by patkub** [comment]
This is already fixed :+1:
